lib/timeline.js

```diff
@@ -164,6 +192,41 @@ Timeline.prototype.unsubscribe = function (userToken, topic, callback) {
   this._request(opts, callback);
 };
 
+/**
+ * List user subscriptions
+ *
+ * @param {String} userToken
+ * @callback {callback} callback
+ */
+Timeline.prototype.listSubscriptions = function (userToken, callback) {
+  if (typeof userToken !== 'string') {
+    return process.nextTick(function () {
+      callback(new Error('Expected userToken to be a string.'));
+    });
+  }
+
+  var opts = {
+    method: 'GET',
+    endpoint: '/v1/user/subscriptions',
+    headers: {
+      'X-User-Token': userToken
+    }
+  };
+
+  this._request(opts, function (err, body, res) {
+    if (err) {
+      return callback(err);
+    }
+
+    try {
+      var json = JSON.parse(body);
+      callback(null, json.topics, res);
+    } catch (e) {
+      callback(new Error('Could not parse API response for listSubscriptions.'));
```

Comment:
this is the double callback bug: when executing a userland callback which throws an error, the callback is executed again. callbacks should only ever be executed at most once, unless otherwise very explicitly specified (this is very very unusual and if you find this is the case then there's probably a better pattern to use like streams)

to fix this, wrap only our library code (JSON.parse) with the try/catch.

```js
var json;
try { json = JSON.parse(body); }
catch (err) {
  return callback(new Error(..));
}

callback(null, json);